### PR TITLE
SAAS-477 Add read_more_url to check results

### PR DESCRIPTION
**Pull request by palash25**
[SAAS-477](https://jira.percona.com/browse/SAAS-477)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1460

- [ ] Links to other linked pull requests (optional).
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -14,11 +14,11 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "release/2.15"
+  branch = "SAAS-477-add-link-to-checks"
```
Comment (palash25) [inline Gopkg.toml:17]: todo

**Review by Dasio — APPROVED** [review]

**Comment by jprukner** [comment]
@palash25 I don't feel qualified to approve this PR, but I did not find anything fundamentally wrong with the code.
There are conflicts though.

**Comment by palash25** [comment]
> @palash25 I don't feel qualified to approve this PR, but I did not find anything fundamentally wrong with the code.
There are conflicts though.

thats fine @jprukner thanks for reviewing

**Review by BupycHuk — APPROVED** [review]

File: Gopkg.toml [hunk 2]
```diff
@@ -14,11 +14,11 @@ required = [
...
 
 [[constraint]]
   name = "github.com/percona-platform/saas"
-  branch = "main"
+  branch = "SAAS-477-add-link-to-checks"
```
Comment (palash25) [inline Gopkg.toml:21]: todo

**Review by artemgavrilov — APPROVED** [review]

File: Gopkg.toml (hunk 2 above)
Comment (artemgavrilov) [reply Gopkg.toml]: Done

File: Gopkg.toml (hunk 1 above)
Comment (artemgavrilov) [reply Gopkg.toml]: Done

**Pull request merged**